t/page/download.rb

```diff
@@ -23,7 +23,7 @@
       end
 
       it 'should be at rawgit' do
-        subject.download_url.must_include 'cdn.rawgit.com'
+        subject.download_url.must_equal index_at_known_sha.index_url
```

Comment:
I think you're right about removing this, it's essentially an alias for `index.index_url`, so I don't think there's any benefit to be had in keeping this test. I'll get rid of it.